mrp_bom_structure_xlsx_level_1/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+MRP BOM Structure Report XLSX Level 1
+=====================================
+
+This module was written to extend the MRP capabilities of Odoo, and
+allows users to export the BOM structure to MS Excel .XLSX format, but up to
```

Comment:
allows users to export up to level 1 the BOM structure in MS Excel .XLSX format.

(I would not explain more)